tests/test_cp_als.py

```diff
@@ -206,3 +206,28 @@ def test_cp_als_sptensor_zeros(capsys):
     capsys.readouterr()
     assert pytest.approx(output3["fit"], 1) == 0
     assert output3["normresidual"] == 0
+
+
+@pytest.mark.indevelopment
+def test_cp_als_tensor_printitn(capsys, sample_tensor):
+    (data, T) = sample_tensor
+
+    # default printitn
+    (M, Minit, output) = ttb.cp_als(T, 2, printitn=1)
+    capsys.readouterr()
+    assert pytest.approx(output["fit"]) == 1
```

Comment:
We don't actually care about the fit here so can we ignore checking it? As far as I can tell this is just a smoke test.
In which case it might also be nice to set `maxiters` to something small to save the CPU cycles.